packages/date-picker/src/DatePicker/components/DateInput/utils/getDescription.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+
+export const getDescription = (
+  description: React.ReactNode
+): React.ReactNode => {
+  switch (typeof description) {
+    case "string":
+      if (description === "") return "Format: mm/dd/yyyy"
+      return `${description} (Format: mm/dd/yyyy)`
+    case "object":
```

Comment:
You could possible use `React.isValidElement` here https://reactjs.org/docs/react-api.html#isvalidelement

Reply:
Hmm... I just tested it out, and the only time we'd have a discrepancy is when someone might attach a `ReactPortal` as the value (is not a valid element, but is an object). Realistically though, no one would ever do that, so it's not really something to worry about.

Makes sense to me, and reads a little clearer :) Will push up the change.